test/parameter_test.dart

```diff
@@ -20,8 +20,48 @@ void main() {
 class ParameterTest extends DartdocTestBase {
   @override
   String get libraryName => 'parameters';
-  @override
-  String get sdkConstraint => '>=2.17.0 <3.0.0';
+
+  void test_formalParameter_referenced() async {
+    var library = await bootPackageWithLibrary('''
+/// Text [p].
+void f(int p) {}
+''');
+    var f = library.functions.named('f');
+    // There is no link, but also no wrong link or crash.
+    expect(f.documentationAsHtml, '<p>Text <code>p</code>.</p>');
+  }
+
+  void test_formalParameter_referenced_notShadowedElement() async {
+    var library = await bootPackageWithLibrary('''
+/// Text [p].
+void f(int p) {}
+var p = 0;
+''');
+    var f = library.functions.named('f');
+    // There is no link, but also no wrong link or crash.
+    expect(f.documentationAsHtml, '<p>Text <code>p</code>.</p>');
+  }
+
+  void test_formalParameter_referenced_notShadowedPrefix() async {
+    var library = await bootPackageWithLibrary('''
+import 'dart:async' as p;
+/// Text [p].
+void f(int p) {}
+''');
+    var f = library.functions.named('f');
+    // There is no link, but also no wrong link or crash.
+    expect(f.documentationAsHtml, '<p>Text <code>p</code>.</p>');
+  }
+
+  void test_formalParameter_referenced_wildcard() async {
+    var library = await bootPackageWithLibrary('''
+/// Text [_].
+void f(int _) {}
+''');
+    var f = library.functions.named('f');
+    // There is no link, but also no wrong link or crash.
+    expect(f.documentationAsHtml, '<p>Text <code>_</code>.</p>');
+  }
 
   void test_formalParameter_generic_method() async {
```

Comment:
Test suggestion: `this._`/`super._` in class constructors?

Reply:
Good idea, done!